src/common/components/WriteLayout/style.css.ts

```diff
@@ -0,0 +1,44 @@
+import { css } from "@styled-system/css";
+
+export const stepBtn = css ({
+    margin: '1rem',
+    display: 'flex',
+    justifyContent: 'space-evenly',
+    paddingBottom: '1.62rem',
+    gap: '1rem',
+})
+
+export const entireLayout = css ({
+    display: 'flex',
+    flexDirection: 'column',
+    height: '100vh'
```

Comment:
```suggestion
    height: '100%'
```
Safe Area를 적용해줬기 때문에 `100vh`로 설정하면 핸드폰에 따라 화면이 넘칠 수도 있을거 같아요